sentry_flink/sentry_flink/flink/flink_adapter.py

```diff
@@ -45,6 +57,22 @@
 T = TypeVar("T")
 
 
+class RoutingFunction(ProcessFunction):
```

Comment:
Flink-specific wrappers like this live in the flink_translator. Can we move it there? Similar to `FlinkAggregate`. I also like to prepend the class name with `Flink` because it's clear that it's a runtime-specific operation.